src/main/java/uk/gov/hmcts/reform/civil/service/citizenui/DashboardClaimInfoService.java

```diff
@@ -43,12 +45,22 @@ public List<DashboardClaimInfo> getClaimsForDefendant(String authorisation, Stri
     }
 
     private List<DashboardClaimInfo> getCases(String authorisation) {
-        Query query = new Query(QueryBuilders.matchAllQuery(), emptyList(), 0);
-        SearchResult claims = coreCaseDataService.searchCases(query, authorisation);
-        if (claims.getTotal() == 0) {
-            return Collections.emptyList();
-        }
-        return translateSearchResultToDashboardItems(claims);
+        List<DashboardClaimInfo> dashboardClaimItems = new ArrayList<>();
+
+        int batchSize = BATCH_SIZE; // fetch 100 cases per batch
```

Comment:
is it approved by PO that we show all 100 cases in dashboard? what happens when we have more than 100 cases?

Reply:
I changed any way because just doesn't work and we don't needed the BATCH_SIZE